venues/ICLR.cc/2018/Conference/process/submissionFastProcess.js

```diff
@@ -0,0 +1,216 @@
+function() {
```

Comment:
this should be the submissionProcess.js because it has the comment invitations. And then you should commit the submissionFastProcess.js without the comment invitations.